Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -789,13 +783,13 @@
 #define ENABLE_NOTIFY_FILTERING 1
 #endif
 
-#if !defined(ENABLE_APP_PRIVACY_REPORT) && PLATFORM(IOS_FAMILY) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 150000
+#if !defined(ENABLE_APP_PRIVACY_REPORT) && PLATFORM(IOS_FAMILY)
```

Comment:
This one is weird, I don't know where `PLATFORM(IOS_FAMILY) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 150000` is true. I think that this may exclude watchOS and tvOS, but maybe not visionOS?

Reply:
It seems likely that whoever wrote this just made a mistake, and meant to enable this feature on the 2021-aligned OSes.

Reply:
I think you're right, that, to make sure these patches don't have behavior changes, I should exclude any OSes which don't set `__IPHONE_OS_VERSION_MIN_REQUIRED` which I believe is just watchOS and tvOS.

Reply:
I think that they set it to some frozen value that's smaller than 150000. And visionOS probably sets it to 170000, but I'm just guessing. As I said, this is a weird one.